Makefile

```diff
@@ -52,17 +54,43 @@ get: $(foreach SVC, $(GET_SVCS), get.$(SVC))
 
 # Start environment
 .PHONY: up
-up: get vendor/hosts
+up: up/basic
 	@$(foreach SVC, $(START_SVCS), $(shell docker-compose -f services/$(SVC)/docker-compose.yml up -d))
+	@echo "Full NeoFS Developer Environment is ready"
+
+# Build up NeoFS
+.PHONY: up/basic
+up/basic: get vendor/hosts
+	@$(foreach SVC, $(START_BASIC), $(shell docker-compose -f services/$(SVC)/docker-compose.yml up -d))
 	@./bin/tick.sh
 	@./bin/config.sh string SystemDNS container
-	@echo "NeoFS Developer Environment is ready"
+	@echo "Basic NeoFS Developer Environment is ready"
+
+# Build up certain service
+.PHONY: up/%
+up/%: get vendor/hosts
+	@docker-compose -f services/$*/docker-compose.yml up -d
+	@echo "Developer Environment for $* service is ready"
 
 # Stop environment
 .PHONY: down
-down:
+down: down/add down/basic
+	@echo "Full NeoFS Developer Environment is down"
+
+.PHONY: down/add
+down/add:
 	$(foreach SVC, $(STOP_SVCS), $(shell docker-compose -f services/$(SVC)/docker-compose.yml down))
 
+# Stop basic environment
+.PHONY: down/basic
+down/basic:
```

Comment:
@realloc Don't see any use case in `down/basic` target. Actually, I prefer previous solution from @EliChin with two simple target: shutdown everything or shutdown specific service.

Reply:
The purpose of this target is to keep the previous `make down` behaviour without breaking the service stop order.